views/src/main/java/com/lush/view/LushEditText.java

```diff
@@ -117,7 +118,7 @@ private void formatViews(Context context)
 		exclamationLayoutParams.height = (int) height;
 		exclamationLayoutParams.width = (int) height;
 		redExclamationImage.setLayoutParams(exclamationLayoutParams);
-		editText.setId(Calendar.getInstance().get (Calendar.MILLISECOND));
+		editText.setId(Calendar.getInstance().get(Calendar.MILLISECOND));
```

Comment:
Is it not going to cause issues by setting the ID after it's already been set when inflating the View hierarchy? Or is this constructor not called in that instance?

Reply:
This was a previous change which is only formatted in this change. The reason for this is: Apparently the ID was not properly set when inflating the the View (either not set or set the same for all the edittexts in used in a screen). It was causing all fields to have the same text after screen rotation in screens with multiple edit texts. This ensures that they have unique ID and fixes the issue.